app/serializers/dossier_serializer.rb

```diff
@@ -38,4 +38,8 @@ def accompagnateurs
   def invites
     object.invites_gestionnaires.pluck(:email)
   end
+
+  def commentaires
+    object.commentaires.order(created_at: :asc)
```

Comment:
Utiliser le scope `ordered` de Commentaire

Voire mieux : mettre ce scope en default scope (un peu plus long mais worth it?)